### PR TITLE
AD-11: Adding factorial file

### PR DESCRIPTION
**Pull request by bhagavansprasad**
Can you please review my code

### CODE REVIEW

File: bhagavan/mypython/factorial.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+def factorial(n):
+    print("n :", n)
+    
+    f = 1
+    for i in range(1, n+1):
+        f = f * i
+    return f
+
+n = 5
+t = factorial(n)
```
Comment (abhiroyals) [inline bhagavan/mypython/factorial.py]: add a block in main functionality.

File: bhagavan/mypython/factorial.py [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+print(t)
```
Comment (shivasaimarapakala) [inline bhagavan/mypython/factorial.py]: the print used here is not supported by python3

Comment (bhagavansprasad) [reply bhagavan/mypython/factorial.py]: ACCEPTED: Changes are applied

File: bhagavan/mypython/factorial.py (hunk 1 above)
Comment (bhagavansprasad) [reply bhagavan/mypython/factorial.py]: ACCEPTED: Changes are applied

**Review by abhiroyals — COMMENTED** [review]
Resolved

**Review by shivasaimarapakala — APPROVED** [review]

**Review by abhiroyals — APPROVED** [review]

**Pull request merged**